image-inspector.go

```diff
@@ -295,6 +341,24 @@ func main() {
 
 	supportedVersions := APIVersions{Versions: []string{VERSION_TAG}}
 
+	inspectorMetadata := NewInspectorMetadata(imageMetadata)
+
+	var arf_body []byte
+	if inspectorOptions.Oscap {
+		inspectorMetadata.OpenSCAP.Status = StatusSuccess
+		log.Printf("OpenSCAP scanning %s", inspectorOptions.DstPath)
+		err = openscap.OscapScan(inspectorOptions.DstPath, OSCAP_RESULT_DIR)
+		if err != nil {
+			inspectorMetadata.OpenSCAP.setError(err)
+			fmt.Printf("Unable to run OpenSCAP: %v\n", err)
+		}
+		arf_body, err = ioutil.ReadFile(path.Join(OSCAP_RESULT_DIR, openscap.ARF_RESULT_FILE))
```

Comment:
@enoodle we discussed this. What if you're running two `image-inspector` at the same time? They would both try to write to the same file. We have to make this temporary file unique.

Check if you can create it beforehand (with an unique name) and using [O_EXCL](http://linux.die.net/man/3/open).